### ISSUE TITLE
Agent status report

### ISSUE THREAD

**Issue by jesuslinares** [wazuh/wazuh-kibana-app#596]
It would be great if the app could send reports like this by email:

![image](https://user-images.githubusercontent.com/1523318/41653422-4d82dfe8-7486-11e8-969e-3846f60f494f.png)

**Comment by jesusgn90** [comment]
Right, once the Wazuh API give us the "unique" routes we'll add some functionality to agents in the Wazuh App. Second step is to study cons about e-mail integration but I think it's not a problem cause Node.js has high support for these kind of tasks. 

Regards,
Jesús

**Comment by TaLoN1x** [comment]
Any plans for this functionality?
